x-pack/plugins/infra/public/pages/metrics/hosts/components/table/filter_action.tsx

```diff
@@ -32,32 +36,52 @@ export const FilterAction = ({ selectedItemsCount, filterSelectedHosts }: Filter
   };
 
   return (
-    <EuiPopover
-      isOpen={isPopoverOpen}
-      closePopover={closePopover}
-      data-test-subj="bulkAction"
-      panelPaddingSize="s"
-      button={
+    <Container>
+      <EuiPopover
+        isOpen={isPopoverOpen}
+        closePopover={closePopover}
+        data-test-subj="bulkAction"
+        panelPaddingSize="s"
+        className={selectedItemsCount === 0 ? '' : 'is-visible'}
+        css={css`
+          position: absolute;
+          opacity: 0;
+          visibility: hidden;
+          &:is(.is-visible) {
```

Comment:
Instead of adding a class to control the CSS you can use a condition inside the `css` param: 
```
        css={css`
          position: absolute;
          opacity: 0;
          visibility: hidden;
          ${selectedItemsCount > 0 &&
          `
            opacity: 1;
            ${euiCanAnimate} {
              transition: opacity ${euiTheme.animation.extraFast} ease-in;
            }
            visibility: visible;
          `}
        `}
```

Reply:
I think this approach would generate new classnames whenever the condition changes. I found an approach more consistent with what `emotion/css` provides: https://emotion.sh/docs/@emotion/css#cx

Reply:
This approach is also good 👍